public/js/track.js

```diff
@@ -261,10 +274,28 @@ function addPlaylistEventListeners() {
                         $('#playlistModal').modal('hide');
                         window.open(url, "_blank");
                     })
-                    .catch((errMessage) => {
-                        alert(errMessage);
+                    .catch((error) => {
+
+                        $('#errorAlert').append('<a class="close alert-close">&times</a>' +
+                            '<span>'+'Sorry, we encountered an internal server error. ' + '</span>');
```

Comment:
What do you think about, `Sorry, we encountered an error`? It's like a bit less technical